ctrtool/utils.h

```diff
@@ -41,7 +41,12 @@ int makedir(const char* dir);
 
 u64 _fsize(const char *filename);
 
-#ifndef _WIN32
+#ifdef _WIN32
+inline int fseeko64(FILE *__stream, long long __off, int __whence)
```

Comment:
This breaks compiling on MinGW-w64. Please add a ifdef, so it will only be evaluated on Visual Studio.